src/main/webapp/project/NewProjectDocuments.js

```diff
@@ -1,7 +1,8 @@
 import { Component, Fragment } from 'react';
 import { WizardStep } from '../components/WizardStep';
-import { hh, h, h1, span, button, div } from 'react-hyperscript-helpers';
+import { hh, h, h1, span, button, div, p, small } from 'react-hyperscript-helpers';
```

Comment:
Not related to this PR, but I'd like folks' opinions on if `react-hyperscript-helpers` has been a time saver or not ... I'm sort of on the fence about this. Will ask in standup.